components/aftership/actions/common.mjs

```diff
@@ -0,0 +1,234 @@
+import aftership from "../aftership.app.mjs";
+
+export default {
+  methods: {
+    getData() {
```

Comment:
Common files should be located in a `common` folder:
`/aftership/actions/common/common.mjs`